examples/ping_service.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+"""
+This script calls the ping method for a Thrift service in the given
+host and port. By default, it uses the TBinary protocol.
+
+```
+$ examples/ping_service.py 127.0.0.1 9090
+Received: method=ping, type=reply, seqid=0, header=None, fields=fields=[(string, 0, None)]
+...
+
+"""
+
+from __future__ import print_function
```

Comment:
This isn't needed if this is Python 3-only (which is what the shebang indicates).

Reply:
Yeah actually, we should probably start dropping the old py2 workarounds & compat tricks.